README.template.md

```diff
@@ -59,39 +59,42 @@ There are a number of `docker-compose.yml` files provided by this repository:
 
 ## Override
 
-This repository ignores `docker-compose.override.yml` which will be included in
-any `docker compose` commands you invoke by default.
+This git repository does not track `docker-compose.override.yml` which will 
+be included by default in all `docker compose` commands you invoke.
 
 Two platform dependent templates that allow for access to the hosts `SSH agent`
 are provided for `development` environments.
 
-Simply copy the appropriate `docker-compose.PLATFORM.yml` file into
+Simply copy the appropriate* `docker-compose.PLATFORM.yml` file into
```

Comment:
Which one is appropriate? If I'm running Mac or Windows, do I choose darwin or linux?

Reply:
Darwin is Mac and linux for Linux but there doesn't seem to be an example for windows.

Reply:
Indeed. I don't think most new Mac users know they're on "Darwin" so I'll add a comment to that effect.